desc/compute/_profiles.py

```diff
@@ -607,6 +607,97 @@ def _iota_rr(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="iota_current",
+    label="\\iota_{\\mathrm{current}}",
+    units="~",
+    units_long="None",
+    description="Rotational transform (normalized by 2pi), current contribution",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="r",
+    data=["iota_den", "iota_current_num"],
+)
+def _iota_current(params, transforms, profiles, data, **kwargs):
+    data["iota_current"] = transforms["grid"].replace_at_axis(
+        data["iota_current_num"] / data["iota_den"], 0
+    )
```

Comment:
I misled you earlier. For this limit to be 0, we would need to make the invalid assumption that $\partial_{\rho \rho} I = 0$ at axis. We can only assume the first derivative is 0 at axis.

The limit of this one should be `alpha_r / iota_den_r` where `alpha_r` is the same thing as `iota_current_num_r`. See equation 48 in the document here: https://github.com/PlasmaControl/DESC/pull/556#issue-1773649567. It's the first term in the numerator divided by the denominator.

Reply:
where is this document?

Reply:
It's at the top of the description of Github Pull request 558. titled math to compute rotational transform in desc